_posts/2022-11-30-extend-clang-to-resugar-template-specialization-accesses.md

```diff
@@ -120,7 +118,7 @@ increase my confidence as a developer in the LLVM open community!
 
 **Contact me!**
 
-Email: mizvekov@gmail.com
+Email: <mizvekov@gmail.com>
```

Comment:
This will create a link for the email, which will open the user's default mail client with the specified "to" address. I think this will improve the overall user experience.

I can revert these changes if you want.

Reply:
I think we use this as an ID in other places and I'd prefer to not have that feature.

Reply:
Okay I'll revert these changes for emails.

Reply:
I think it would be enough if we go to the profile of the person instead of composing an email...

Reply:
ping.

Reply:
Okay I'll make that change.